.travis.yml

```diff
@@ -0,0 +1,11 @@
+language: ruby
+rvm:
+- 2.1.5
+
+before_install:
+- "export DISPLAY=:99.0"
```

Comment:
Actually heroku was throwing an error on #to_h before I set rvm to 2.1.5 in the travis file. Weird I know...